packages/server/package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "adm-zip": "0.5.9",
+    "async-batch": "1.1.2",
```

Comment:
Presumably this is from the other pr to add `async-batch` and could be dropped from this one?

Reply:
oh yeah, looks like a commit from Aditya's pr got merged into my branch somehow. I'll remove it.